lib/good-influx-http.js

```diff
@@ -1,23 +1,70 @@
 'use strict';
 
-const Hoek = require('hoek');
-const GoodHttp = require('good-http');
-const Wreck = require('wreck');
+const Hoek = require('@hapi/hoek');
+const Stream = require('stream');
+const Wreck = require('@hapi/wreck');
+const Os = require('os');
 const LineProtocol = require('./line-protocol.js');
 
 const internals = {
     defaults: {
+        threshold: 20,
+        errorThreshold: 0,
+        schema: 'good-influx',
+        wreck: {
+            timeout: 60000,
+            headers: {}
+        },
+        host: Os.hostname(),
         metadata: false
     }
 };
 
-class GoodInfluxHttp extends GoodHttp {
+class GoodInfluxHttp extends Stream.Writable {
     constructor(endpoint, config, schemas) {
-        super(endpoint, config, schemas);
-        this._settings.schema = 'good-influx';
+        config = config || {};
+        const settings = Object.assign({}, internals.defaults, config);
+
+        if (settings.errorThreshold === null) {
+            settings.errorThreshold = -Infinity;
+        }
+
+        super({ objectMode: true, decodeStrings: false });
```

Comment:
This does not seem to depend on the code above it. Seems like you could move the `super` to the first line of the constructor.

Reply:
fixed